### PR TITLE
libfuse/libdokan: enable teams for admins on prod

### PR DESCRIPTION
**Pull request by strib**
And enable them for all users on staging.

(To be merged after we deploy prod.)

### CODE REVIEW

File: libdokan/fs.go [hunk 1]
```diff
@@ -377,11 +377,11 @@ func (f *FS) open(ctx context.Context, oc *openContext, ps []string) (dokan.File
 			return nil, false, dokan.ErrAccessDenied
 		}
 		return f.root.private.open(ctx, oc, ps[1:])
-	case `TEAM` == ps[0]:
+	case strings.ToUpper(TeamName) == ps[0]:
```
Comment (jzila) [inline libdokan/fs.go]: Do you think the Go compiler will optimize out this `ToUpper` call on a constant?

**Comment by strib** [comment]
> Not mentioned, but this also enables teams by default for all staging/devel users.

Not mentioned in the subject, but it's mentioned on the first line of the PR description.  (It was already enabled for all devel users before this PR.)

File: libdokan/fs.go (hunk 1 above)
Comment (strib) [inline libdokan/fs.go]: Dunno!  I was just making it consistent with the other places above.  I think this one happened concurrently with switchover to `ToUpper`.  I'm not worried about performance here, but if you are we can revert to using constants in a separate PR.

Comment (jzila) [inline libdokan/fs.go]: Naw it's fine

**Pull request merged**